cmd/initialize/cmd.go

```diff
@@ -65,15 +65,15 @@ func init() {
 		&args.dlt,
 		"delete-stack",
 		false,
-		"Deletes stack template applied to your AWS account during the 'init' command.\n",
+		"Deletes stack template applied to your AWS account during the 'init' command.",
 	)
 	flags.MarkDeprecated("delete-stack", "use --delete instead")
 
 	flags.BoolVar(
 		&args.dlt,
 		"delete",
 		false,
-		"",
+		"Deletes stack template applied to your AWS account during the 'init' command.",
```

Comment:
Couldn't really find more info on this command, would you say we can copy the deprecated usage?

@vkareh

Reply:
Ah sorry, I misunderstood your question.

Yes, you can copy the message. They do the same thing.